sdk/communication/communication-common/CHANGELOG.md

```diff
@@ -4,6 +4,16 @@
 
 ### Features Added
 
+- Added support for a new communication identifier `TeamsExtensionUserIdentifier`.
+    - Added a type `TeamsExtensionUserKind`.
+    - Added a method `isTeamsExtensionUserIdentifier` to check if the identifier is `TeamsExtensionUserIdentifier`.
+    - Mandatory fields of `TeamsExtensionUserIdentifier` are `userId`, `tenantId` and `resourceId`.
+    - With this version, rawId starting with `8:acs` may be either `CommunicationUserIdentifier` or new `TeamsExtensionUserIdentifier`.
```

Comment:
may be we could just mention that `8:acs:{resourceId}_{tenantId}_{userId}` will represent the new identifier